src/text-area/index.ts

```diff
@@ -47,6 +53,7 @@ export interface TextareaProperties
 	placeholder?: string;
 	value?: string;
 	onClick?(value: string): void;
+	onValidate?: (valid: boolean | undefined) => void;
```

Comment:
does this not need `valid?: { valid?: boolean; message?: string } | boolean;` plus logic for displaying it?

Reply:
Yup, you are right. Will add that.